v4_MIGRATION_GUIDE.md

```diff
@@ -488,7 +527,7 @@ public class PayPalActivity extends AppCompatActivity {
       payPalClient.onBrowserSwitchResult(browserSwitchResult, (payPalAccountNonce, error) -> {
         if (payPalAccountNonce != null) {
           // Send nonce to server
-          String nonce = payPalNonce.getString();
+          String payPalAccountNonce = payPalNonce.getString();
```

Comment:
```suggestion
          String nonce = payPalAccountNonce.getString();
```
I think `payPalNonce` was the typo right?

Reply:
Yikes yeah thanks for that.